settings.py

```diff
@@ -24,5 +24,9 @@
 USE_TZ = False
```

Comment:
Consider setting `USE_TZ = True` to enable timezone support. This will help handle datetime objects correctly and avoid potential issues related to timezones.